shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/API/Validation.hs

```diff
@@ -16,7 +16,7 @@ module Shelley.Spec.Ledger.API.Validation
   )
 where
 
-import Byron.Spec.Ledger.Core (Relation (..))
+import Shelley.Spec.Ledger.Core (Relation (..))
```

Comment:
Wait, what? Why?

These names mean something to consumers of this library, eg me. 

Was this named incorrectly? If so fine, but the commit message should state that.

Reply:
Furthermore at least half of PR comment above belongs in the git comment so that the reasons for the change are recorded with the commit itself rather than with the merge commit. Everything other than future plans should be in the commit comment.

Reply:
> Was this named incorrectly? If so fine, but the commit message should state that.

Yes, the plan is to divorce this module from Byron so that we can implement the functions the way we want to (ie efficiently) without having to change Byron. I had thought this was clear from the PR message. I like the idea of putting these notes in the commit message!

Reply:
To be clear, the Byron module `Byron.Spec.Ledger.Core` hasn't moved or changed. It has been copied and renamed for Shelley as a baby step so that we can dramatically strip it down and remove things that we don't want. (such as the `Ord` constraint on `Relation` so that we don't require the _values_ of a map to have `Ord`, etc).